tests/migration_test.go

```diff
@@ -1766,11 +1766,22 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 						Expect(err).ToNot(HaveOccurred(), "Should successfully get new VMI")
 						vmiPod := tests.GetRunningPodByVirtualMachineInstance(newvmi, tests.NamespaceTestDefault)
 
-						if newvmi.Status.Interfaces[0].IP == vmiPod.Status.PodIP {
-							return nil
+						if newvmi.Status.Interfaces[0].IP != vmiPod.Status.PodIP {
+							return fmt.Errorf("IP field doesnt match PodIP")
 						}
 
-						return fmt.Errorf("interface not updated with latest IP")
+						if len(newvmi.Status.Interfaces[0].IPs) != len(vmiPod.Status.PodIPs) ||
+							len(newvmi.Status.Interfaces[0].IPs) == 0 {
+							return fmt.Errorf("Status.Interfaces len is zero or not equal PodIPs len")
+						}
+
+						for i, ip := range vmiPod.Status.PodIPs {
```

Comment:
I think you'll end up making this comparison in plenty other places. 

Could you expose this as a function ?

Reply:
done